<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Auto Load</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolButton" name="startButton">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="Experiment.qrc">
             <normaloff>:/openicon/icons/media-playback-start-7.png</normaloff>:/openicon/icons/media-playback-start-7.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="stopButton">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="Experiment.qrc">
             <normaloff>:/openicon/icons/media-playback-stop-7.png</normaloff>:/openicon/icons/media-playback-stop-7.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="statusLabel">
         <property name="text">
          <string>Idle</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="elapsedLabel">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>26</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0:00.0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="TableViewExtended" name="historyTableView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="1">
        <widget class="QSpinBox" name="ovenChannelBox">
         <property name="toolTip">
          <string>oven enable digital out</string>
         </property>
         <property name="minimum">
          <number>-31</number>
         </property>
         <property name="maximum">
          <number>31</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Shutter</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSpinBox" name="counterChannelBox">
         <property name="toolTip">
          <string>counter channel</string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>7</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Oven</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Counter</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Th. bare</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="MagnitudeSpinBox" name="maxTimeBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>maximum oven on time</string>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="MagnitudeSpinBox" name="thresholdOvenBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Threshold while oven is on</string>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Th. oven</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="MagnitudeSpinBox" name="thresholdBareBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Threshold with oven off</string>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Max Time</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Laser delay</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="MagnitudeSpinBox" name="laserDelayBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>switch on laser after oven on</string>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="shutterChannelBox">
         <property name="toolTip">
          <string>shuter enable digital out</string>
         </property>
         <property name="minimum">
          <number>-31</number>
         </property>
         <property name="maximum">
          <number>31</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Check time</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="MagnitudeSpinBox" name="checkTimeBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Threshold with oven off</string>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MagnitudeSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>MagnitudeSpinBox</header>
  </customwidget>
  <customwidget>
   <class>TableViewExtended</class>
   <extends>QTableView</extends>
   <header>TableViewExtended</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget_2</tabstop>
  <tabstop>startButton</tabstop>
  <tabstop>stopButton</tabstop>
  <tabstop>thresholdOvenBox</tabstop>
  <tabstop>thresholdBareBox</tabstop>
  <tabstop>checkTimeBox</tabstop>
  <tabstop>maxTimeBox</tabstop>
  <tabstop>laserDelayBox</tabstop>
  <tabstop>ovenChannelBox</tabstop>
  <tabstop>shutterChannelBox</tabstop>
  <tabstop>counterChannelBox</tabstop>
  <tabstop>historyTableView</tabstop>
 </tabstops>
 <resources>
  <include location="Experiment.qrc"/>
 </resources>
 <connections/>
</ui>
